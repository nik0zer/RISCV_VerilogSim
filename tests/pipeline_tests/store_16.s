.section .text
.global _start

_start:
    addi x1, x0, 1
    addi x5, x0, 7
    sd x1, 0x10(x0)
    sd x1, 0x18(x0)
    sb x5, 0x10(x0)
    ld x2, 0x10(x0)
    ld x2, 0x18(x0)
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop
    nop