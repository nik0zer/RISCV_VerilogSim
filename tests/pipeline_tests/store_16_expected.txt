x
x
x
x
0000000000000001
0000000000000007
x
x
x
x
0000000000000007
0000000000000007
0000000000000000
0000000000000000
0000000000000000
0000000000000000
0000000000000000
0000000000000000
0000000000000000
0000000000000000
0000000000000000
0000000000000000